README.md

```diff
@@ -0,0 +1,184 @@
+>>> _ = t.gettext # create alias for method
+>>>
+>>> print _('exit') # 'exit' key's Spanish translated value will be used
+Adiós
+>>>
+```
+
+Obtaining language/locale codes
+-------------------------------
+This package requires that valid (BCP47 compliant) language/locale codes be provided when asked; for example, when calling `GPClient.translation()` (see [Examples](#examples)). From these codes, the language, region, and script subtags will be extracted.
+
+Some example codes are:
+
+* zh-Hans
+* pt-BR
+* ja
+* en_US
+
+There are several ways to get the code for the working locale. One way
+is to use the [locale module](https://docs.python.org/2/library/locale.html) (avaliable as part of the Python standard library).
+
+```python
+>>> import locale
+>>> myLocale = locale.getdefaultlocale()
+>>> print myLocale
+('en_US', 'UTF-8')
+>>> code = myLocale[0]
+>>> print code
+en_US
+```
+
+For the above example, the language code is `en_US` - where `en` is the language subtag, and `US` is the region subtag.
+
+Generating documentation
+------------------------
+Documentation can be generated using [Sphinx](http://www.sphinx-doc.org).
+
+You must first install it:
+
+```shell
+$ pip install sphinx
+```
+
+Then, to auto generate the documentation, run:
+
+```shell
+$ cd $BASEDIR/docs
+$ make clean
+$ make html
+```
+
+To navigate the documentation, open `$BASEDIR/docs/_build/html/index.html`.
+
+Creating distribution package
+-----------------------------
+First update `CHANGES.txt` and `setup.py` if necessary (e.g. update version number), then create the preferred [distribution package](http://python-packaging-user-guide.readthedocs.org/en/latest/distributing/#packaging-your-project).
+
+**Wheel distribution** (Recommended)
+
+```shell
+$ pip install wheel
+$ python setup.py bdist_wheel
+```
+
+**Source distribution**
+
+```shell
+$ python setup.py sdist
+```
+
+Note: Source distribution contains tests as well.
+
+**Build distribution**
+
+```shell
+$ python setup.py bdist
+```
+
+The new distribution files should be located under `$BASEDIR/dist/`.
+
+Publishing to PyPI (internal only)
+----------------------------------
+PyPI requires that the README file be in [reStructuredText](https://en.wikipedia.org/wiki/ReStructuredText) format, but this README file is written in Markdown. Therefore, before publishing to PyPI, you must use [Pandoc](http://pandoc.org/) to convert the Markdown file to reStructuredText.
+
+Use the following command:
+
+```shell
+$ pandoc --from markdown --to rst README.md -o README.rst
+```
+
+Running Tests
+-------------
+Refer to [test/README.md](./test/README.md).
+
+End Notes
+----------
+You are most welcome to [submit issues](https://github.com/IBM-Bluemix/gp-python-client/issues), or [fork the repository](https://github.com/IBM-Bluemix/gp-python-client).
+
+`gp-python-client` is published under the [Apache License Version 2.0](https://github.com/IBM-Bluemix/gp-python-client/blob/master/LICENSE.txt).
```

Comment:
might just replace this with the `LICENSE` section at the end of  https://raw.githubusercontent.com/IBM-Bluemix/gp-js-client/master/README.md